packages/katello/katello/hostname-change.rb

```diff
@@ -213,6 +226,10 @@ def setup_opt_parser
           @options[:confirm] = confirm
         end
 
+        opt.on("--skip-dns", "skip updating DNS records") do |skip_dns|
```

Comment:
I think this should come with some kind of warning text.

If foreman is configured to manage DNS but is not able to pass the pre-check, or this flag is given, something is probably really wrong. Although I suggested adding this flag, I think there could be a very good argument to not provide it. 

@ekohl - any input on this?

Reply:
I would advocate for keeping it, I'm sure we could rationalize that there would be no way it should be needed, but I've found in actual user environments it helps to have an way to workaround issues even if its just for debugging purposes or if someone has a very opinionated "special" setup and will do the updating themselves. However, I think we should be very explicit about what the option is and when to use it.

The helper text could be more specific about when it would be applicable. `"skip updating DNS records even if they are managed by #{@default_program}".` I wonder if we should only show the option if `@scenario_answers['foreman_proxy']['dns']` is truthy as well?

And a confirmation prompt as @jturel suggested if they use that option and have dns managed. A buy-in `y/n` does help us sleep better at night putting in something that is not the recommended happy path.

Reply:
If we do keep it @johnpmitsch I will update the message per your suggestion, thanks.

Holding off for now on "keep the flag?" and "add the confirm?" until I hear more opinions.

Reply:
+1 keep +1 add confirmation with a sternly worded message that they're going into uncharted waters preferably mentioning some of the things that could fail to work after the script completes

Reply:
@johnpmitsch Message updated
```
options
    -u, --username username          admin username (required)
    -p, --password password          admin password (required)
    -y, --assumeyes                  answer yes for all questions
        --skip-dns                   skip updating DNS records even if they are managed by satellite
    -h, --help                       help
```